osutil/group.go

```diff
@@ -63,6 +63,34 @@ func lookupGroup(groupname string) (string, error) {
 	return strconv.Itoa(int(grp.gr_gid)), nil
 }
 
+// Use implementation from:
+// https://golang.org/src/os/user/cgo_lookup_unix.go
```

Comment:
could we take this one step further and pull in the `Group` type?

Reply:
you mean this one? 
```
// from https://golang.org/src/os/user/user.go
type Group struct {
	Gid  string // group ID
	Name string // group name
}
```

Reply:
yes

Reply:
I pulled in `os/user.Group` as suggested. `FindGroupOwning()` was updated to return `*Group, error`, other calls were returning either gid or name, so I left those intact.